charts/oes/values.yaml

```diff
@@ -624,6 +630,38 @@ ldap:
   groupRoleAttributes: cn
   userDnPattern: cn={0},dc=example,dc=org
 
+#####################################################
+# SAML AUthentication
+#####################################################
+# saml.jks and metadata.xml are to be mounted as secrets
+# with names oes-saml-jks and oes-saml-metadata resp.
```

Comment:
Let's give a provision to user to simply specify the name of the secret in which saml.jks and oes-sam-metadata are present and handle the stuff in the templates in such a way that files from that particular secret will be mounted on the necessary pods

Reply:
Fixed